src/dune_rules/ocaml_flags.mli

```diff
@@ -4,6 +4,13 @@ open! Import
 
 type t
 
+module Keywords : sig
+  val empty : string option * string list
```

Comment:
Looking at this interface, this would definitely benefit from a `Keywords.t` type to avoid the repetition of `string option * string list`.